test/fixtures/http-response/features/steps/steps.rb

```diff
@@ -0,0 +1,3 @@
+When('I ignore invalid {word}') do |type|
+  Maze.config.captured_invalid_requests -= [type.to_sym]
```

Comment:
It might be a little clearer to use `delete` here (and wherever this is actually going to be used)

```suggestion
  Maze.config.captured_invalid_requests.delete(type.to_sym)
```